CONTRIBUTING.md

```diff
@@ -175,11 +175,11 @@ sure that the cross-compiler can be found. This can be done by adding it to the
 `PATH` environment variable as described in the ARM Embedded Toolchain
 `readme.txt` file:
 
-    export PATH=$PATH:$install_dir/gcc-arm-none-eabi-*/bin
+    export PATH=$PATH:$install_dir/gcc-arm-none-eabi-*/bin/*
```

Comment:
I don't think this should have a trailing `*`, otherwise it would expand to all files in the `bin` directory instead of the `bin` directory itself.

Reply:
I tested again and actually both variants don't work for me, I get
```
bene@linux:~$ cd lego/pybricks-micropython/
bene@linux:~/lego/pybricks-micropython$ poetry shell
Spawning shell within /home/bene/lego/pybricks-micropython/.venv
bene@linux:~/lego/pybricks-micropython$ . /home/bene/lego/pybricks-micropython/.venv/bin/activate
(pybricks-micropython-py3.10) bene@linux:~/lego/pybricks-micropython$ echo $PATH
/home/bene/lego/pybricks-micropython/.venv/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games:/home/bene/.local/bin:/home/bene/.local/bin
(pybricks-micropython-py3.10) bene@linux:~/lego/pybricks-micropython$ export PATH=$PATH:/home/bene/lego/gcc-arm-none-eabi-*/bin
(pybricks-micropython-py3.10) bene@linux:~/lego/pybricks-micropython$ echo $PATH
/home/bene/lego/pybricks-micropython/.venv/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games:/home/bene/.local/bin:/home/bene/.local/bin:/home/bene/lego/gcc-arm-none-eabi-*/bin
(pybricks-micropython-py3.10) bene@linux:~/lego/pybricks-micropython$ make technichub
make[1]: Entering directory '/home/bene/lego/pybricks-micropython/micropython/mpy-cross'
Use make V=1 or set BUILD_VERBOSE in your environment to increase build verbosity.
make[1]: Leaving directory '/home/bene/lego/pybricks-micropython/micropython/mpy-cross'
make[1]: Entering directory '/home/bene/lego/pybricks-micropython/bricks/technichub'
Use make V=1 or set BUILD_VERBOSE in your environment to increase build verbosity.
GEN build/genhdr/qstr.i.last
Traceback (most recent call last):
  File "/home/bene/lego/pybricks-micropython/bricks/technichub/../../micropython/py/makeqstrdefs.py", line 203, in <module>
    preprocess()
  File "/home/bene/lego/pybricks-micropython/bricks/technichub/../../micropython/py/makeqstrdefs.py", line 73, in preprocess
    for output in p.imap(pp(flags), chunks):
  File "/usr/local/lib/python3.10/multiprocessing/pool.py", line 873, in next
    raise value
  File "/usr/local/lib/python3.10/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/home/bene/lego/pybricks-micropython/bricks/technichub/../../micropython/py/makeqstrdefs.py", line 57, in run
    return subprocess.check_output(args.pp + flags + files)
  File "/usr/local/lib/python3.10/subprocess.py", line 421, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/usr/local/lib/python3.10/subprocess.py", line 503, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/usr/local/lib/python3.10/subprocess.py", line 971, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/local/lib/python3.10/subprocess.py", line 1847, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'arm-none-eabi-gcc'
make[1]: *** [../../micropython/py/mkrules.mk:108: build/genhdr/qstr.i.last] Error 1
make[1]: *** Deleting file 'build/genhdr/qstr.i.last'
make[1]: Leaving directory '/home/bene/lego/pybricks-micropython/bricks/technichub'
make: *** [Makefile:92: technichub] Error 2
(pybricks-micropython-py3.10) bene@linux:~/lego/pybricks-micropython$ 
```

for both of them. And since it is compiling after changing the CROSS_COMPILE I did not notice that this change does not help. I guess from a theoretical point of view you are right and it should work with the path, not sure what I am doing wrong.

Reply:
Do we still need this if we use the version provided by apt install?

Reply:
I am using debian bullseye, not sure if there were packages, so I installed it myself. For me just setting CROSS_COMPILE variable is enough.

Reply:
Sounds good. And thanks so much for sponsoring our project! We really appreciate it 😄

Reply:
What is the output of `echo $PATH` after running the export command? I suppose the use of the star in the path could be problematic depending on how the shell program interprets it and if there is more than one Arm toolchain installed.